src/sdk/openmldb_api.cc

```diff
@@ -57,7 +57,7 @@ ParameterRow::ParameterRow(const OpenmldbHandler* handler) : handler_(handler) {
 std::shared_ptr<openmldb::sdk::SQLRequestRow> ParameterRow::get_parameter_row() const {
     sql_parameter_row_ = ::openmldb::sdk::SQLRequestRow::CreateSQLRequestRowFromColumnTypes(parameter_types_);
     sql_parameter_row_->Init(str_length_);
-    for (int i = 0; i < record_.size(); ++i) {
+    for (unsigned long int i = 0; i < record_.size(); ++i) {
```

Comment:
please resolve cpplint warning

Reply:
Thanks for you helpful response, I have converted the for-loop counter's type from `unsigned long int` to `size_t`, in which caused cpplint warnings.